src/app/docs/selection-box/selection-box-docs.component.html

```diff
@@ -0,0 +1,269 @@
+<sky-docs-demo-page
+  moduleName="SkySelectionBoxModule"
+  moduleSourceCodePath="src/app/public/modules/selection-box/"
+  pageTitle="Selection box"
+>
+  <sky-docs-demo-page-summary>
+    <!-- Page summary goes here -->
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    <!-- Demo goes here -->
+  </sky-docs-demo>
+
+  <sky-docs-design-guidelines>
+    
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+    
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p>
+          Use selection boxes when users must make a selection or answer a standalone question before proceeding, usually as part of a setup or configuration task. In these scenarios, the selection boxes should represent the only decision users need to make on a page.
```

Comment:
Does `the selection boxes should represent the only decision users need to make on a page` mean that no other inputs should be on the page? In Tim's mockup for the onboarding process, he has the example you are using here for selecting roles right under three inputs to collect personal info. I can't tell if that violates this guideline.

Reply:
This might be from an older version of his design. The current one follows these guidelines.

Reply:
I'm referring to the most recent prototypes. I talked to Tim on Friday and he provided a link to the prototypes. He has 2 pages in the onboarding workflow that could use selection boxes, and the first is the roles stuff that you mimic in your image. On the screen for that, the section to select roles is preceded by a section with three inputs for personal information.
![image](https://user-images.githubusercontent.com/7850105/112988978-71e29880-9132-11eb-9004-83b69a173349.png)